riotfile.py

```diff
@@ -313,11 +325,21 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="ddtracerun",
             command="pytest {cmdargs} --no-cov tests/commands/test_runner.py",
-            pys=select_pys(),
-            pkgs={
-                "redis": latest,
-                "gevent": latest,
-            },
+            venvs=[
+                Venv(
+                    pys=select_pys(max_version="3.10"),
+                    pkgs={
+                        "redis": latest,
+                        "gevent": latest,
+                    },
+                ),
+                Venv(
```

Comment:
Worth having a comment on why we split the suite (cause of gevent).

Also, since we have a bunch of gevent work-arounds for 3.11 can we add a common marker so that when gevent adds support for 3.11 we can easily find all the places we need to update? Maybe something like `FIXME[gevent-3.11]: ...`?